docs/Running_Rinkeby_Nitro_Node.md

```diff
@@ -80,9 +82,9 @@ Note: If you’re interested in accessing the Arbitrum Rinkeby network but you d
   The arb-relay is in the same docker image.
 - Here is an example of how to run nitro-relay for Rinkeby:
   ```
-  docker run --rm -it  -p 0.0.0.0:9642:9642 --entrypoint relay offchainlabs/nitro-node:v2.0.0-beta.7-340b812 --node.feed.input.url wss://rinkeby.arbitrum.io/feed --l2.chain-id=421611
+  docker run --rm -it  -p 0.0.0.0:9642:9642 --entrypoint relay offchainlabs/nitro-node:v2.0.0-beta.8-5ed2c72 --node.feed.input.url wss://rinkeby.arbitrum.io/feed --l2.chain-id=421611`
```

Comment:
I don't think we want `--l2.chain-id` for the relay, and there's a trailing backtick

Reply:
:+1: